spec/controllers/idv/ssn_controller_spec.rb

```diff
@@ -187,14 +192,6 @@
           expect(subject.idv_session.applicant).to be_blank
         end
       end
-
-      it 'does not change threatmetrix_session_id when updating ssn' do
-        subject.idv_session.ssn = ssn
-        put :update, params: params
-        session_id = subject.idv_session.threatmetrix_session_id
-        subject.threatmetrix_view_variables
-        expect(subject.idv_session.threatmetrix_session_id).to eq(session_id)
-      end
```

Comment:
This spec is deleted because it wasn't actually testing anything - session_id was always nil, and update doesn't call threatmetrix_view_variables unless the SSN is invalid in update. Which doesn't happen unless someone sends a curl request or specifically turns off javascript on this page, because there's on-page validation of the SSN. Added a better spec under #show, above. This also applies to in_person/ssn_controller_spec.rb.